.classpath

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This change does not seem to be related to the enhancement proposed by this PR 🤔